docs/README.md

```diff
@@ -0,0 +1,288 @@
+["seed_nonce_revelation", "delegation", "transaction", "activate_account", "origination", "reveal", "double_endorsement_evidence", "double_baking_evidence", "endorsement"]
+```
+
+#### Attribute Values with Prefix
+
+It is possible to get values for high-cardinality properties with a prefix. In this example we get all `accounts.manager` values starting with "KT1".
+
+```bash
+curl --request GET --header 'apiKey: hooman' --url 'https://conseil-dev.cryptonomic-infra.tech:443/v2/metadata/tezos/alphanet/accounts/manager/KT1'
+```
+
+```json
+...
+```
+
+#### Extended metadata
+
+As mentioned in the [Datasource configuration](#datasource-configuration) section, it's possible to provide additional details for the items Conseil manages. 
+
+### Tezos Chain Data Query
+
+Data requests in Conseil v2 API are sent via `POST` command and have a prefix of `/v2/data/<platform>/<network>/<entity>`. The items in curly braces match up to the `name` property of the related metadata results. There additional header to be set in the request: `Content-Type: application/json`.
+
+The most basic query has the following JSON structure.
+
+```json
+{
+	"fields": [],
+	"predicates": [],
+	"limit": 5
+}
+```
+
+This will return 5 items without any filtering for some entity. Note that the query does not specify which entity is being requested. If this is sent to `/v2/data/tezos/alphanet/blocks`, it will return five rows with all the fields available in the blocks entity.
+
+```bash
+curl -d '{ "fields": [], "predicates": [], "limit": 5 }' -H 'Content-Type: application/json' -H 'apiKey: <API key>' -X POST '<conseil-host>/v2/data/tezos/alphanet/blocks/'
+```
+
+The query grammar allows for some fine-grained data extraction.
+
+#### `fields`
+
+It is possible to narrow down the fields returned in the result set by listing just the ones that are necessary. The `fields` property of the query is an array of `string`, these values are the `name`s from the `/v2/metadata/<platform>/<network>/<entity>/attributes/` metadata response.
+
+#### `predicates`
+
+`predicates` is an array containing objects. The inner object as several properties:
+- `field` – attribute name from the metadata response.
+- `operation` – one of: in, between, like, lt, gt, eq, startsWith, endsWith, before, after.
+- `set` – an array of values to compare against. `in` requires two or more elements, `between` must have exactly two, the rest of the operations require a single element.
+- `inverse` – boolean, setting it to `true` applied a `NOT` to the operator
+- `precision` – for numeric field matches this specifies the number of decimal places to round to.
+
+#### `limit`
+
+Specifies the maximum number of records to return.
+
+#### `orderBy`
+
+Sort condition, multiple may be supplied for a single query. Inner obejct(s) will contain `field` and `direction` properties. The former is `name` from the `/v2/metadata/<platform>/<network>/<entity>/attributes/` metadata response, the latter is one of 'asc', 'desc'.
```

Comment:
I see an small inconsistency in how keywords are emphasised like the predicate operation enumerations, which are plain text, the same used as an example for the predicates' set field, and here with the ordering values in single quotes.
Using a single consistent notation might help the reader

Reply:
this applies to the aggregation's functions